### DISCUSSION TITLE
Git segment working on WSL2 but not in a Windows shared folder

### DISCUSSION THREAD

**Discussion by maxboaventura** [JanDeDobbeleer/oh-my-posh#2754] [Troubleshoot] [answered]
Today I've tested this amazing tool for the first time, downloading the latest version on my WSL2 Ubuntu, but the Git segment is not working outside the WSL drive. If I change to a folder in the /mnt path the Git segment is missing. I usually have a few things placed in my Windows user Document folder and would love to see it working for things placed over there.

Checking the code on _git_test.go_ I presume that the issue is related to tests checking the SharedDrive.
env.On("InWSLSharedDrive").Return(false)

**Comment by JanDeDobbeleer** [comment, marked as answer]
In the shared drive, it will use git.exe as calling linux executables is slow in that context. This probably means that you don't have git installed in Windows. To validate that assumption, you can run `oh-my-posh debug` in that location.
